sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.institutions.controller.ts

```diff
@@ -113,6 +113,29 @@ export class EducationProgramInstitutionsController extends BaseController {
     );
   }
 
+  /**
+   * Allows a program to be deactivated.
+   * @param programId program to be deactivated.
+   * @param payload information to support the deactivation.
+   */
+  @ApiNotFoundResponse({
+    description: "Not able to find the education program.",
+  })
+  @ApiUnprocessableEntityResponse({
+    description: "The education program is already set as requested.",
+  })
```

Comment:
dont we need to check if the institution have the particular location access? to deactivate the program?

Reply:
Programs are Institution level only 😉